src/libs/styled/useColorScheme.tsx

```diff
@@ -1,38 +1,55 @@
-import { ColorSchemeName, Appearance } from 'react-native'
+import { useColorScheme as useSystemColorScheme } from 'react-native'
 
+import { useFeatureFlag } from 'libs/firebase/firestore/featureFlags/useFeatureFlag'
+import { RemoteStoreFeatureFlags } from 'libs/firebase/firestore/types'
 import { createStore } from 'libs/store/createStore'
 
-export type ColorScheme = 'dark' | 'light'
+export enum ColorSchemeEnum {
+  LIGHT = 'light',
+  DARK = 'dark',
+  SYSTEM = 'system',
+}
 
-type State = { colorScheme: ColorScheme }
+export type ColorScheme = ColorSchemeEnum.LIGHT | ColorSchemeEnum.DARK
+export type ColorSchemeFull = ColorSchemeEnum
 
-const DEFAULT_COLOR_SCHEME = 'light'
+type State = { colorScheme: ColorSchemeFull }
 
-const defaultState: State = {
-  colorScheme: DEFAULT_COLOR_SCHEME,
-}
+const DEFAULT_COLOR_SCHEME: ColorSchemeFull = ColorSchemeEnum.LIGHT
+const defaultState: State = { colorScheme: DEFAULT_COLOR_SCHEME }
```

Comment:
cette valeur n'est plus réutilisée, on pourrait l'inliner

```suggestion
const defaultState: State = { colorScheme: ColorSchemeEnum.LIGHT }
```